src/cli.ts

```diff
@@ -43,8 +43,7 @@ const argDefs = [
   {
     name: 'outDir',
     type: String,
-    description:
-        'Type declarations output directory (default concatenated to stdout).',
+    description: 'Type declarations output directory (required).',
```

Comment:
Since you don't have a default option and this is required, consider maybe making outDir `defaultOption: true` so `gen-typescripts-declarations ./typings`?

Reply:
Oh cool, didn't know about that feature. I think I'll keep it required explicit though, because you might confuse the default with `root`.